hypershift-operator/controllers/hostedcluster/hostedcluster_controller.go

```diff
@@ -4066,14 +4077,9 @@ func isUpgradeable(hcluster *hyperv1.HostedCluster, releaseImage *releaseinfo.Re
 		return true, "", nil
 	}
 
-	upgradeImage, exists := hcluster.Annotations[hyperv1.ForceUpgradeToAnnotation]
-	if !exists {
-		return true, "", fmt.Errorf("cluster version is not upgradeable")
-	} else if upgradeImage != hcluster.Spec.Release.Image {
-		return true, "", fmt.Errorf("cluster version is not upgradeable, force annotation is present but does not match desired release image")
-	} else {
-		return true, "cluster version is not upgradeable, upgrade is forced by annotation", nil
-	}
+	// Upgradeable is false and no exception criteria were met, cluster is not upgradable
+	return true, "", fmt.Errorf("cluster version is not upgradeable")
```

Comment:
should this be false?

Reply:
In the original code, we did
```
return true, "", fmt.Errorf("cluster version is not upgradeable")
```

I think this first return value is not really `bool indicating whether the HostedCluster is upgrading` but rather `bool indicating whether hcluster.Status.Version.Desired.Image != hcluster.Spec.Release.Image`

Reply:
ack, thx